samples/export.ts

```diff
@@ -0,0 +1,19 @@
+export const numberRegexp: string;
+
+export interface StringValidator {
+    isAcceptable(s: string): boolean;
+}
+
+export namespace Hoge {
+    export class Fuga {
+        name: string
+    }
+}
+
+declare namespace PIXI {
+    const VERSION: string;
+}
+
+declare module "pixi.js" {
+    export = PIXI;
+}
```

Comment:
Ah! You removed one EOL too many. Now there's this ugly "No newline at end-of-file" mark. Could you add back exactly one newline here so that git/GitHub don't do this, please?

Sorry. I should have anticipated that this could happen and warn against that effect in my previous comment :s

Reply:
Fixed in https://github.com/sjrd/scala-js-ts-importer/pull/54/commits/685eef3d725b08de74e27682b00054327b2583c1